core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -743,8 +742,8 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
 
     sendRecords(producers.head, 10, topicPartition)
     var result = client.deleteRecords(Map(topicPartition -> RecordsToDelete.beforeOffset(5L)).asJava)
-    var lowWatermark = result.lowWatermarks().get(topicPartition).get().lowWatermark()
-    assertEquals(5L, lowWatermark)
+    var lowWatermark = Option(result.lowWatermarks.get(topicPartition).get.lowWatermark)
```

Comment:
nit: Could we use `Some`? Same below.

Reply:
I initially used Some(..) but subsequently learned that Some(null) is not None, however Option(null) is None and should be preferred so thought that it is better to use Option here. But I really just read about this today, so this might be misguided :)

Reply:
Yeah, that makes sense, but in this case `lowWatermark` is a `long` which cannot be null.

Reply:
Fair point, I'll push a commit with this fixed and @ijuma 's concern addressed.